docs/content/en/docs/Components/istio.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Integrate Istio with Kurator"
+linkTitle: "Integrate Istio with Kurator"
+description: >
+    Integrate Istio with Kurator?
+---
+
+### What is Istio
+
+[Istio](https://istio.io) is an open source service mesh that layers transparently onto existing distributed applications.
```

Comment:
This is not a right sentence.

Istio is an open source service mesh that simplifies observability, traffic management, security, and policy for distributed applications.

Reply:
Istio is a service mesh—a modernized service networking layer that provides a transparent and language-independent way to flexibly and easily automate application network functions.